openquake/qa_tests_data/gmf_ebrisk/case_4/expected/loss_curves-quantile-0.85.csv

```diff
@@ -0,0 +1,4 @@
+#,,,,"kind='quantile-0.85', risk_investigation_time=50.0"
+asset_id,loss_type,loss_value,loss_ratio,period
```

Comment:
`period` → `return_period` for the field names to be consistent with existing outputs.

Reply:
The `event_based_risk` and `ebrisk` calculators also have an `annual_frequency_of_exceedence` field.

Reply:
Fixed